lib/xilinx/xc7series/frames.cc

```diff
@@ -0,0 +1,106 @@
+#include <fstream>
+#include <iostream>
+
+#include <prjxray/xilinx/xc7series/ecc.h>
+#include <prjxray/xilinx/xc7series/frames.h>
+
+namespace prjxray {
+namespace xilinx {
+namespace xc7series {
+
+Frames::Frames2Data& Frames::getFrames() {
+	return frames_data_;
+}
+
+// Reads the contents of the frames file and populates
+// the Frames container.
+int Frames::readFrames(const std::string& frm_file_str) {
```

Comment:
Method level comments should be at the interface (e.g. header file) rather than implementation.  If a comment is required to understand the function/method, they should be in the body of the function/method.

Reply:
Ok, I will move all the comments to the header files.